database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -13,7 +13,7 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->increments('id');
+            $table->increments('id')->unsigned();
```

Comment:
auto increments are by default unsigned,
you will need to add unsigned only to foreign keys